openlibrary/templates/books/add.html

```diff
@@ -71,7 +71,7 @@ <h1>$_("Add a book to Open Library")</h1>
         $if recaptcha and not ctx.user:
             <div class="formElement">
                 <div class="label">$_("Since you are not logged in, please type in the text or number(s) below.")</div>
```

Comment:
I think this messaging is no longer correct with v2, we will need to find something more appropriate. Sometimes it is just a checkbox, other times you need to identify multiple images.